### PR TITLE
[#3863] Add an non overwrite option to BrinkFilesystem.copyFolder.

### PR DESCRIPTION
**Pull request by brunogola**
Scope
=====

BrinkFilesystem?.copyFolder should have an option for copying without overwriting files. 

Changes
=======

Added an `overwrite` argument to the `copyFolder` method, it defaults to `True` keeping the old behavior.

How to try and test the changes
===============================

reviewers: @adiroiban 

Please check if changes make sense.

### CODE REVIEW

File: brink/filesystem.py [hunk 1]
```diff
@@ -169,11 +170,16 @@ def copyFolder(
...
+
+                destination_file = self.join([destination_folder, file_])
+
+                if not overwrite and os.path.exists(destination_file):
+                    skip_file = True
+
                 if skip_file:
                     continue
 
                 source_file = self.join([source_folder, file_])
```
Comment (adiroiban) [inline brink/filesystem.py]: maybe also move the source_file

**Comment by brunogola** [comment]
needs-review

**Comment by adiroiban** [comment]
looks good. see my tests and changes.

merge them if the pass pqm

thanks

changes-approved

**Comment by adiroiban** [comment]
looks good. thanks!

**Pull request closed without merging**